deployment/ansible/development/roles/pycharm/templates/run-manager-environment.xml.j2

```diff
@@ -5,7 +5,7 @@
         <env name="DATABASE_USERNAME" value="docker" />
         <env name="DATABASE_PASSWORD" value="docker" />
         <env name="DATABASE_HOST" value="db" />
-        <env name="INASAFE_REALTIME_BROKER_URL" value="amqp://guest:guest@{{ interpreters.inasafe_realtime.ipaddress }}:{{ docker_port_forward.inasafe_realtime.rabbitmq }}/" />
+        <env name="INASAFE_REALTIME_BROKER_URL" value="amqp://guest:guest@{{ inasafe_realtime.broker }}:{{ inasafe_realtime.broker_port }}/" />
```

Comment:
I hae a problem in this one, I think if you are using docker image production, we need to set it to the default one, `amqp://guest:guest@rabbitmq:5672/`